lib/lightning_web/live/workflow_live/edit.ex

```diff
@@ -1314,7 +1314,7 @@ defmodule LightningWeb.WorkflowLive.Edit do
     <div
       id={"selected-template-label-#{@template.id}"}
       phx-mounted={fade_in()}
-      class="absolute z-[9999] m-4 opacity-75 bg-white/100 border border-gray-200 rounded-lg p-6"
+      class="absolute z-40 m-4 opacity-75 bg-white/100 border border-gray-200 rounded-lg p-6"
```

Comment:
@elias-ba I updated this to z-40. is there a reason z-[9999] was used? if not then I think z-40 should be fine.